llvm/test/CodeGen/X86/pr65895.ll

```diff
@@ -21,10 +21,11 @@ define i32 @PR65895() {
 ; CHECK-NEXT:    # =>This Inner Loop Header: Depth=1
 ; CHECK-NEXT:    jmp .LBB0_2
 ; CHECK-NEXT:  .LBB0_3: # %for.end
+; CHECK-NEXT:    movzbl %al, %ecx
```

Comment:
To address this can you add the OneUse checks back to the x86 version of `isDesirableToCommuteWithShift`?

Reply:
done